contracts/pods/PodFactory.sol

```diff
@@ -135,85 +148,49 @@ contract PodFactory is CoreRef, IPodFactory {
         return podController.isTransferLocked(podId);
     }
 
-    //////////////////// STATE-CHANGING API ////////////////////
-
-    /// @notice Create a child Orca pod with optimistic timelock. Callable by the DAO and the Tribal Council
-    ///         Returns podId, optimistic pod timelock address and the Pod Gnosis Safe address
-    ///         This will lock membership transfers by default
-    function createChildOptimisticPod(PodConfig calldata _config)
-        public
+    /// @notice Deploy the genesis pod, one time use method
+    function deployGenesisPod(PodConfig calldata _config)
```

Comment:
Want a simple way to deploy the first pod. Could do this in the constructor but it references constructor params that I want to be immutable, and would prefer to keep them immutable. 

This method is intended to be used to deploy the TribalCouncil, before the DAO vote happens, so we already know the podId and can't get bricked. A seperate call in the DAO vote must be made to lock transfers, as at deploy time the factory doesn't have the necessary role to call into the `PodAdminGateway`.

Reply:
ok thats fine but can you just call it council instead of genesis. Don't want it to be confused with the protocol launch

Reply:
Also there is no access control on this method but I think thats ok

Reply:
membership transfer lock is not enabled by default on the first pod

Reply:
Will rename to council. It will be called as part of the deployment script so haven't included access control. 

Membership transfer lock is not called in the deploy function as the podFactory will not yet have the relevant roles granted to it by the DAO, to be able to call into the `podAdminGateway`.

Flow is:
**Deploy script**
1. Deploy `podFactory`. It will not yet have any roles
2. Call `deployCouncilPod()` - to deploy the initial TribalCouncil. It will be unlocked

**DAO vote**
1. Grant `podFactory` the relevant roles, e.g. `POD_ADMIN`
2. DAO calls the locking function to lock transfers on the TribalCouncil. It is included in `fip_82.ts` and tested for